Podcast/GridSeries.swift

```diff
@@ -17,21 +17,19 @@ class GridSeries: NSObject {
     var largeArtworkImageURL: URL?
     var smallArtworkImageURL: URL?
     var isSubscribed: Bool
-    var lastUpdated: Date
```

Comment:
Why are we removing lastUpdated from a GridSeries? This is what we use for cells in Subscription View (which has text of the last updated episode)

Reply:
If we removed lastUpdated from here we will also need to change SeriesGridCollectionViewCell to reflect these changes (rn it is currently giving an error ... for now we can comment out and stub in a TODO)